catroid/src/main/java/org/catrobat/catroid/ui/fragment/BaseProjectListFragment.java

```diff
@@ -128,7 +127,7 @@ public int compare(ProjectData project1, ProjectData project2) {
 			}
 		});
 
-		projectAdapter = new ProjectListAdapter(getActivity(), R.layout.list_item, projectList);
+		projectAdapter = new ProjectListAdapter(context, R.layout.list_item, projectList);
```

Comment:
`context` is null at this time, this causes the app to crash at older Android versions, changing it back to `getActivity()` works.

Reply:
Ok, I see. However, I can't rely on getActivity() here, because CopyPhiroProjectsTask in the ProjectListFragment of the Phiro-flavor causes the Fragment to be detached from it's activity. So, the context/activity must be remembered. 
So, I'm sure the issue is that, because we are not using the SupportFragmentManager, onAttach(Context) is simply not called. 
I think the best approach would be to override onAttach(Context) & onAttach(Activity) for API < 23, as described here: https://stackoverflow.com/questions/32077086/android-onattachcontext-not-called-for-api-23